Update on the ONNX Runtime documentation (#567)
huggingface/optimum: docs/source/onnxruntime/usage_guides/pipelines.mdx

[{'label': 'POSITIVE', 'score': 0.9998838901519775}]
```

_Note: The default models used in the [`~pipelines.pipeline`] function are not optimized for inference or quantized, so there won't be a performance improvement compared to their PyTorch counterparts._

### Using vanilla Transformers model and converting to ONNX

The [`~pipelines.pipeline`] function accepts any supported model from the [Hugging Face Hub](https://huggingface.co/models).
There are tags on the Model Hub that allow you to filter for a model you'd like to use for your task.

<Tip>

To be able to load the model with the ONNX Runtime backend, the export to ONNX needs

> to be supported for the architecture considered.

Reviewer: ```suggestion
to be supported for the considered architecture.
```

> You can check the list of the supported architectures

Reviewer: ```suggestion
You can check the list of supported architectures
```

> [here](/exporters/onnx/package_reference/configuration#Supported_architectures).

Reviewer: ```suggestion
[here](/exporters/onnx/package_reference/configuration#supported-architectures).
```

## Optimizing and quantizing in pipelines

The [`~pipelines.pipeline`] function can not only run inference on vanilla ONNX Runtime checkpoints - you can also use
checkpoints optimized with the [`~optimum.onnxruntime.ORTQuantizer`] and the [`~optimum.onnxruntime.ORTOptimizer`].

> Below you can find two examples on how you could the [`~optimum.onnxruntime.ORTOptimizer`] and the

Reviewer: ```suggestion
Below you can find two examples of how you could use the [`~optimum.onnxruntime.ORTOptimizer`] and the
```
